### ISSUE TITLE
Can't get screen styles to work with Swing Screen

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Valyrox/lanterna#13]
```
When using putString with a Swing Screen, the styles like Bold and Underline 
don't seem to get applied -- all the text is normal.  They work fine when I 
swap the Swing screen for a console one.

I'm using OS X 10.7.4 in case that matters.
```

Original issue reported on code.google.com by `st...@stevelosh.com` on 7 Jul 2012 at 4:29

**Comment by GoogleCodeExporter** [comment]
```
Yes, this is an interesting one, many terminal emulators doesn't actually make 
`CSI 1 m` text bold, instead the just increase the brightness of the colour. 
Underline isn't widely supported either. But that being said, SwingTerminal 
doesn't support underlined text due to laziness on my part and supporting bold 
text shouldn't be too hard either.


```

Original comment by `mab...@gmail.com` on 9 Jul 2012 at 1:45
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
I've just committed a fix for this on 2.0.x-branch and deployed an updated 
snapshot to Maven (2.0.1-SNAPSHOT). Please give it a try!
```

Original comment by `mab...@gmail.com` on 9 Jul 2012 at 12:24
- Changed state: **Fixed**
